src/software/world/robot_state.h

```diff
@@ -56,6 +56,48 @@ class RobotState
      */
     AngularVelocity angularVelocity() const;
 
+    /**
+     * Returns if the ball is in the break beam
+     *
+     * @return ball in break beam
+     */
+    bool ballInBeam() const;
+
+    /**
+     * Update ball in break beam
+     *
+     * @param new ball in break beam
```

Comment:
Nit:

Maybe add the underscores to the variable name, or change to 'new break beam status'.

This reads like a normal sentence first hand which is mildly confusing.

Reply:
good catch